examples/example_metadata/table_inplace_volumes.yml

```diff
@@ -52,7 +52,7 @@ display:
   name: geogrid
 file:
   absolute_path: /some/absolute/path/
-  checksum_md5: 173c646133f5a47c9bfbc4432a0a2812
+  checksum_md5: 0cb778f91e6074c471b4bbce1f919165
```

Comment:
👀 ?

Reply:
Is this calculated on the fly, or is this a static example value?